### PR TITLE
Add provisioned device overview to zwave js

### PR DESCRIPTION
**Pull request by bramkragten**
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/data/zwave_js.ts [hunk 1]
```diff
@@ -350,6 +369,19 @@ export const provisionZwaveSmartStartNode = (
     planned_provisioning_entry,
   });
 
+export const unprovisionZwaveSmartStartNode = (
+  hass: HomeAssistant,
+  entry_id: string,
+  dsk?: string,
+  node_id?: number
```
Comment (balloob) [inline src/data/zwave_js.ts:376]: Why is node ID optional? What happens if you leave it out?

Comment (bramkragten) [reply src/data/zwave_js.ts:376]: (Not sure if the device stays in the provisioning list after it is added to HA?!)

File: src/panels/config/integrations/integration-panels/zwave_js/zwave_js-provisioned.ts [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+        .route=${this.route}
+        .tabs=${configTabs}
+        .columns=${this._columns(this.narrow)}
+        .data=${this._provisioningEntries}
+      >
+      </hass-tabs-subpage-data-table>
+    `;
+  }
+
+  private _columns = memoizeOne(
```
Comment (balloob) [inline src/panels/config/integrations/integration-panels/zwave_js/zwave_js-provisioned.ts:43]: Can we show the date when a device was provisioned ?

Comment (bramkragten) [reply src/panels/config/integrations/integration-panels/zwave_js/zwave_js-provisioned.ts:43]: If the backend adds that info to the provisioning entry, yes

**Review by balloob — APPROVED** [review]

File: src/panels/config/integrations/integration-panels/zwave_js/zwave_js-provisioned.ts (hunk 2 above)
Comment (balloob) [reply src/panels/config/integrations/integration-panels/zwave_js/zwave_js-provisioned.ts:43]: Right so not this PR.

@raman325 / @MartinHjelmare is this something that we can add?

Comment (MartinHjelmare) [reply src/panels/config/integrations/integration-panels/zwave_js/zwave_js-provisioned.ts:43]: The driver doesn't provide a date as far as I know. If we add it separately we'd need to keep track of this state somehow.

Comment (raman325) [reply src/panels/config/integrations/integration-panels/zwave_js/zwave_js-provisioned.ts:43]: that's correct, maybe something we can request as a feature to zwave-js

Comment (balloob) [reply src/panels/config/integrations/integration-panels/zwave_js/zwave_js-provisioned.ts:43]: If we are requesting features, maybe also allow the user to set a name for when it comes online?

Comment (bramkragten) [reply src/panels/config/integrations/integration-panels/zwave_js/zwave_js-provisioned.ts:43]: I think we can add anything we want to the provisioning entry?

https://github.com/zwave-js/node-zwave-js/blob/6ce51e47d7cd6d4419c0be4eb9fc676f38f0b088/packages/zwave-js/src/lib/controller/Inclusion.ts#L141

**Pull request merged**